packages/vscode-extension/e2e_tests/main.ts

```diff
@@ -9,16 +10,14 @@ function setSetting(file: string, variable: RegExp, value: string) {
   fs.writeFileSync(file, result);
 }
 
-function updateSettingsFile(port: number, password: string) {
-  const settingsPath = path.join(
-    __dirname,
-    '../../e2e_tests/fixtures/.vscode/',
-  );
-  const settingsTemplate = path.join(settingsPath, 'settings-template.json');
-  const settingsFile = path.join(settingsPath, 'settings.json');
-  fs.copyFileSync(settingsTemplate, settingsFile);
-  setSetting(settingsFile, /\{PORT\}/g, port.toString());
-  setSetting(settingsFile, /\{PASSWORD\}/g, password);
+function updateDotenvFile(port: number, password: string) {
```

Comment:
changing from settings.json to dotenv so that we can load connection details into process.env and consume them when setting up the test suites